content/docs/aws/authn-authz.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "Authentication and Authorization"
+description = "Authentication and Authorization support for Kubeflow in AWS"
+weight = 90
++++
+
+This section shows the how to setup Kubeflow with Authentization and Authorization support in Amazon Web Services (AWS).
+
+
+## Manifest
+
+In order to simply your setups, we highly recommend you to use this manifest.
+[kfctl_aws_cognito.yaml](https://github.com/kubeflow/manifests/blob/master/kfdef/kfctl_aws_cognito.yaml)
+
+## Traffic Flow
+External Traffic → [ Ingress → Istio ingress gateway → Istio virtual services ]
+
+When you generate and apply kubernetes resources, an ingress is created to manage external traffic to Kubernetes services. The AWS Appliction Load Balancer(ALB) Ingress Controller will provision an Application Load balancer for that ingress. By default, TLS and authentication are not enabled at creation time.
+
+In Kubeflow 0.6 release, community already move from [Ambassador](https://www.getambassador.io/) to [Istio](https://istio.io/) to manage internal traffic. In AWS solution, TLS, authentication can be done at the ALB and and authorization can be done at Istio layer.
```

Comment:
Do we need to mention Kubeflow 0.6 here? Instead, we can just say something like:

`Kubeflow uses [Istio](https://istio.io/) to manage internal traffic.`

Reply:
Make sense. Remove legacy contents.